astropy_helpers/version_helpers.py

```diff
@@ -358,11 +346,6 @@ def generate_version_py(packagename=None, version=None, release=None, debug=None
             f.write(_get_version_py_str(packagename, version, last_githash,
                                         release, debug, uses_git=uses_git))
 
-        invalidate_caches()
-
-        if version_module:
-            imp.reload(version_module)
```

Comment:
I don't understand why it's safe to delete this.  Isn't this necessary to ensure that the version actually gets imported if there was a previous file already there?

Reply:
Basically this isn't needed anymore because we import the version module with ``import_file`` rather than using ``__import__`` - so we aren't *actually* importing ``astropy.version`` (and therefore ``astropy``) anymore and don't need to reload it.